modular_skyrat/modules/new_cells/code/power_cell.dm

```diff
@@ -0,0 +1,54 @@
+/obj/item/stock_parts/cell/crank
+	name = "crank cell"
+	desc = "Go ahead, wind it up to charge it."
+	icon = 'modular_skyrat/modules/new_cells/icons/power.dmi'
+	icon_state = "crankcell"
+	COOLDOWN_DECLARE(discharge_cooldown)
+
+/obj/item/stock_parts/cell/crank/examine(mob/user)
+	. = ..()
+	. += span_notice("ALT-CLICK to start cranking the cell.")
+
+/obj/item/stock_parts/cell/crank/Initialize(mapload, override_maxcharge)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/item/stock_parts/cell/crank/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/obj/item/stock_parts/cell/crank/process(delta_time)
+	if(!COOLDOWN_FINISHED(src, discharge_cooldown))
+		return
+	COOLDOWN_START(src, discharge_cooldown, 2 SECONDS)
+	use(10)
```

Comment:
This could be a var on the cell, like `var/discharge_amount_per_process` or something like that.